cluster/manifests/skipper/deployment.yaml

```diff
@@ -309,6 +315,11 @@ spec:
           - name: filters
             mountPath: /etc/config/default-filters
 {{ end }}
+{{ if eq .ConfigItems.datadome_enabled "true" }}
+          - mountPath: /etc/skipper/lua
+            name: lua
```

Comment:
Lets use the same field order as for others:
```yaml
- name: skipper-ingress-lua
  mountPath: /etc/skipper/lua
  readOnly: true
```

Reply:
boah you are seriously too picky

Reply:
The names above and below are `filters` and `hostname-credentials` so it does not follow your comment